crates/llms/src/bedrock/embed/mod.rs

```diff
@@ -203,10 +203,14 @@ where
     async fn embed_request(
         &self,
         req: CreateEmbeddingRequest,
-    ) -> Result<CreateEmbeddingResponse, OpenAIError> {
+    ) -> EmbedResult<CreateEmbeddingResponse> {
```

Comment:
Thanks for the PR! It looks like we're returning an OpenAIError on line 90 as well - ideally we would clean up all instances of this with this PR.